client/src/Movies/MovieList.js

```diff
@@ -19,34 +21,40 @@ const MovieList = props => {
   }, []);
   
   return (
-    <div className="movie-list">
-      {movies.map(movie => (
-        <MovieDetails key={movie.id} movie={movie} />
-      ))}
-    </div>
+    
+      <div className="movie-list">
```

Comment:
It looks like you're returning only the movie title and not the whole card here?